Nova documentação de Notificações (feature/new-notifications) (#2426)
mercadopago/devsite-docs: guides/additional-content/your-integrations/ipn.es.md

2. En el menú de la izquierda, selecciona **IPN**, y configura la **URLs de producción** que será utilizada para recibir las notificaciones. También podrás probar si la URL indicada está recibiendo notificaciones correctamente, verificando la solicitud, la respuesta dada por el servidor y la descripción del evento.

![ipn](/images/dashboard/ipn_es_.png)

> NOTE
>
> Nota
>
> En caso de ser necesario identificar múltiples cuentas, agrega el parámetro `?cliente=(nombredelvendedor)` endpoint al final de la URL indicada, para identificar a los vendedores.

3. Selecciona los **eventos** para los cuales deseas recibir las notificaciones, que serán enviadas en formato `json` a través de un `HTTP POST` a la URL especificada anteriormente.  Un evento puede ser cualquier actualización sobre el tópico reportado, incluyendo cambios de status o atributos. Consulta la tabla a continuación para ver qué eventos pueden ser configurados teniendo en cuenta la solución de Mercado Pago integrada y las particularidades de negocio.

| Eventos | Nombre en Tus integraciones | Tópico | Productos asociados |
|---|---|---|---|

> | Creación y actualización de pagos | Pagos | `payment` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout Pro<br>Checkout Bricks<br>Suscripciones<br>----[mla, mlm, mlb]----MP Point------------<br>Wallet Connect |

Reviewer: ```suggestion
| Creación y actualización de pagos | Pagos | `payment` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout Pro<br>Checkout Bricks<br>Suscripciones<br>----[mla, mlm, mlb]----Mercado Pago Point------------<br>Wallet Connect |
```

----[mla, mlm, mlb]----| Finalización, cancelación o errores al procesar intenciones de pago de dispositivos Mercado Pago Point. | Integraciones Point | `point_integration_ipn` | Mercado Pago Point |------------
| Alertas de fraude luego del procesamiento de un pedido | Alertas de fraude | `delivery_cancellation` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout PRO |
| Creación, actualización o cierre de órdenes comerciales |  Órdenes comerciales | `merchant_order` | Checkout Pro<br>Código QR  |
| Apertura de contracargos, cambios de status y modificaciones referentes a las liberaciones de dinero.   |   Contracargos | `chargebacks`  | Checkout Pro<br>Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------ <br>Checkout Bricks |

> > WARNING

Reviewer: ----[mlm, mlb]----
> WARNING

> > WARNING

Reviewer: ```suggestion
----[mlm, mlb]----
> WARNING
```

>
> Importante
>
> En caso de dudas sobre los tópicos a activar o los eventos que serán notificados, consulta la documentación [Información adicional sobre notificaciones](/developers/es/docs/your-integrations/notifications/additional-info). 

## Configuración durante la creación de un pago

Durante el proceso de creación de pagos, es posible configurar la URL de notificación de forma más específica para cada pago, utilizando el campo `notification_url` e implementando un receptor de notificaciones. 

> > WARNING

Reviewer: ```suggestion
----[mla, mlb, mlm]----

> WARNING
```

>
> Importante
>

> > Este método no permite configurar notificaciones para el tópico `point_integration_ipn` utilizando este método. Para activarlo, utiliza la configuración mediante Tus integraciones.

Reviewer: ```suggestion
> Este método no permite configurar notificaciones para el tópico `point_integration_ipn` utilizando este método. Para activarlo, utiliza la configuración mediante Tus integraciones.

------------
```

 
A continuación, explicamos cómo configurar notificaciones IPN al crear un pago usando los SDK.

> 1. En el campo `notification_url`, indica la URL desde la que se recibirán las notificaciones, como se muestra a continuación. Para recibir notificaciones exclusivamente vía IPN y no vía Webhooks, es posible agregar el parámetro `source_news=webhooks` a la `notification_url`. Por ejemplo: "https://www.yourserver.com/notifications?source_news=webhooks".

Reviewer 2: Dejo acá mismo comentario con el link cliqueable

> 1. En el campo `notification_url`, indica la URL desde la que se recibirán las notificaciones, como se muestra a continuación. Para recibir notificaciones exclusivamente vía IPN y no vía Webhooks, es posible agregar el parámetro `source_news=webhooks` a la `notification_url`. Por ejemplo: "https://www.yourserver.com/notifications?source_news=webhooks".

Reviewer: ```suggestion
1. En el campo `notification_url`, indica la URL desde la que se recibirán las notificaciones, como se muestra a continuación. Para recibir notificaciones exclusivamente vía IPN y no vía Webhooks, es posible agregar el parámetro `source_news=webhooks` a la `notification_url`. Por ejemplo: `https://www.yourserver.com/notifications?source_news=webhooks`.
```

# IPN

IPN (Instant Payment Notification) es un mecanismo que permite que tu aplicación reciba notificaciones de Mercado Pago, informando el estado de un determinado pago, contracargo y *merchant order*, a través de una llamada HTTP POST para informar sobre sus transacciones.

> WARNING
>
> Importante
>
> Las notificaciones IPN van a ser descontinuadas. Además, a pesar de recibir el *header* `x-Signature`, no permiten la validación mediante la clave secreta para confirmar que las mismas fueron enviadas por Mercado Pago. Si deseas realizar esta validación, recomendamos migrar a [notificaciones Webhooks](/developers/es/docs/your-integrations/notifications/webhooks).   

Las notificaciones IPN pueden ser configuradas de dos maneras:

| Tipo de configuración | Descripción |
|---|---|
| [Configuración a través de Tus integraciones](/developers/es/docs/your-integrations/notifications/ipn#configuracinatravsdetusintegraciones) | Podrá ser configurada solo una URL de notificación por cuenta (dependiendo de la aplicación, más de una de ellas podrá usar la URL). |

> | [Configuración durante la creación de un pago](/developers/es/docs/your-integrations/notifications/ipn#configuracindurantelacreacindeunpago) | Puede ser realizada a partir del campo `notification_url`. La URL podrá ser diferente para cada objeto o aplicación. |

Reviewer: ```suggestion
| [Configuración durante la creación de un pago](/developers/es/docs/your-integrations/notifications/ipn#configuracindurantelacreacindeunpago) | Puede ser realizada a partir del campo `notification_url`. La URL podrá ser diferente para cada objeto. |
```

En esta documentación explicaremos cómo realizar la configuración necesaria para recibir notificaciones IPN a través de Tus integraciones o al momento de crear pagos, y te mostraremos las acciones necesarias para que Mercado Pago valide que los mensajes fueron recibidos correctamente.

## Configuración a través de Tus integraciones

Puedes configurar notificaciones IPN directamente desde Tus integraciones de manera eficiente y segura.

### Indicar URLs y configurar eventos

Para configurar notificaciones IPN mediante Tus integraciones, es necesario indicar las URLs a las que las mismas serán enviadas y especificar los eventos para los cuales deseas recibirlas. 

> WARNING
>
> Importante
>
> Al configurar notificaciones IPN vía Tus integraciones, estarás configurando la URL y los Eventos de **todas las aplicaciones de tu cuenta de Mercado Pago**. 

> Para confiruar URLs y eventos, sigue el paso a paso a continuación:

Reviewer: ```suggestion
Para configurar URLs y eventos, sigue el paso a paso a continuación:
```

> 1. Ingresa a [Tus Integraciones](/developers/panel/app) y selecciona la aplicación para la que deseas activar las notificaciones. En caso de que aún no hayas creado una aplicación, accede a la [documentación sobre el Panel del Desarrollador](/developers/es/docs/your-integrations/dashboard) y sigue las instrucciones para poder hacerlo.

Reviewer: ```suggestion
1. Ingresa a [Tus Integraciones](/developers/panel/app) y selecciona una de las aplicaciones para activar las notificaciones para toda tu cuenta. En caso de que aún no hayas creado una aplicación, accede a la [documentación sobre el Panel del Desarrollador](/developers/es/docs/your-integrations/dashboard) y sigue las instrucciones para poder hacerlo.
```

2. En el menú de la izquierda, selecciona **IPN**, y configura la **URLs de producción** que será utilizada para recibir las notificaciones. También podrás probar si la URL indicada está recibiendo notificaciones correctamente, verificando la solicitud, la respuesta dada por el servidor y la descripción del evento.

![ipn](/images/dashboard/ipn_es_.png)

> NOTE
>
> Nota
>
> En caso de ser necesario identificar múltiples cuentas, agrega el parámetro `?cliente=(nombredelvendedor)` endpoint al final de la URL indicada, para identificar a los vendedores.

> 3. Selecciona los **eventos** para los cuales deseas recibir las notificaciones, que serán enviadas en formato `json` a través de un `HTTP POST` a la URL especificada anteriormente.  Un evento puede ser cualquier actualización sobre el tópico reportado, incluyendo cambios de status o atributos. Consulta la tabla a continuación para ver qué eventos pueden ser configurados teniendo en cuenta la solución de Mercado Pago integrada y las particularidades de negocio.

Reviewer: ```suggestion
3. Selecciona los **eventos** para los cuales deseas recibir las notificaciones, que serán enviadas en formato `JSON` a través de un `HTTP POST` a la URL especificada anteriormente.  Un evento puede ser cualquier actualización sobre el tópico reportado, incluyendo cambios de status o atributos. Consulta la tabla a continuación para ver qué eventos pueden ser configurados teniendo en cuenta la solución de Mercado Pago integrada y las particularidades de negocio.
```

----[mla, mlm, mlb]----| Finalización, cancelación o errores al procesar intenciones de pago de dispositivos Mercado Pago Point. | Integraciones Point | `point_integration_ipn` | Mercado Pago Point |------------
| Alertas de fraude luego del procesamiento de un pedido | Alertas de fraude | `delivery_cancellation` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout PRO |
| Creación, actualización o cierre de órdenes comerciales |  Órdenes comerciales | `merchant_order` | Checkout Pro<br>Código QR  |
| Apertura de contracargos, cambios de status y modificaciones referentes a las liberaciones de dinero.   |   Contracargos | `chargebacks`  | Checkout Pro<br>Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------ <br>Checkout Bricks |

> WARNING
>
> Importante
>
> En caso de dudas sobre los tópicos a activar o los eventos que serán notificados, consulta la documentación [Información adicional sobre notificaciones](/developers/es/docs/your-integrations/notifications/additional-info). 

## Configuración durante la creación de un pago

> Durante el proceso de creación de pagos, es posible configurar la URL de notificación de forma más específica para cada pago, utilizando el campo `notification_url` e implementando un receptor de notificaciones. 

Reviewer: ```suggestion
Durante el proceso de creación de pagos, preferencias u órdenes presenciales, es posible configurar la URL de notificación de forma más específica para cada pago, utilizando el campo `notification_url` e implementando un receptor de notificaciones. 
```

----[mla, mlb, mlm]----

> WARNING
>
> Importante
>
> Este método no permite configurar notificaciones para el tópico `point_integration_ipn` utilizando este método. Para activarlo, utiliza la [configuración mediante Tus integraciones](/developers/es/docs/your-integrations/notifications/ipn#configuracinatravsdetusintegraciones).

------------
 
A continuación, explicamos cómo configurar notificaciones IPN al crear un pago usando los SDK.

> 1. En el campo `notification_url`, indica la URL desde la que se recibirán las notificaciones, como se muestra a continuación. Para recibir notificaciones exclusivamente vía IPN y no vía Webhooks, es posible agregar el parámetro `source_news=webhooks` a la `notification_url`. Por ejemplo: `https://www.yourserver.com/notifications?source_news=webhooks`.

Reviewer: ```suggestion
1. En el campo `notification_url`, indica la URL desde la que se recibirán las notificaciones, como se muestra a continuación. Para recibir notificaciones exclusivamente vía IPN y no vía Webhooks, es posible agregar el parámetro `source_news=ipn` a la `notification_url`. Por ejemplo: `https://www.yourserver.com/notifications?source_news=ipn`.
```
